### PR TITLE
Improve mime type support

### PR DESCRIPTION
**Pull request by daurnimator**
- Lua mime type should be under application/ not text/
  - Ignore mime type parameters (for now)

### CODE REVIEW

File: src/fengari-web.js [hunk 1]
```diff
@@ -128,20 +128,37 @@ const run_lua_script_tag = function(tag) {
 	}
 };
 
-/* watch for new <script type="text/lua"> tags added to document */
+const contentTypeRegexp = /^(.*?\/.*?)([\t ]*;.*)?$/;
+const maybe_run_lua_script_tag = function(tag) {
```
Comment (giann) [inline src/fengari-web.js]: kind of a mouthful, also no indication on what are the criteria for the _maybe_

Comment (daurnimator) [reply src/fengari-web.js]: Got a suggestion for a new name?

Comment (giann) [reply src/fengari-web.js]: `run_lua_mime_script_tag` ?

Comment (daurnimator) [reply src/fengari-web.js]: Is that any better? The function checks if the HTML element is a suitable script tag, and if so: runs it.

Comment (giann) [reply src/fengari-web.js]: then the _maybe_ is unnecessary: only `lua_script_tag` will be run

Comment (daurnimator) [reply src/fengari-web.js]: But it might not be run in some circumstances.
i.e. it's "maybe run the lua code in this tag"

Comment (giann) [reply src/fengari-web.js]: what circumstances ?

Comment (daurnimator) [reply src/fengari-web.js]: That it's a script tag of a suitable mime type. Perhaps later we'll add extra conditions.

**Pull request merged**